src/components/SaveToPhone.js

```diff
@@ -24,20 +30,16 @@ class SaveToPhone extends React.Component {
   };
 
   handleOnSubmit = () => {
-    let page;
+    let path = "";
     // Needed for gatsby builds that don't have references to window
     if (typeof window !== "undefined") {
-      page = window.location.href;
+      path = window.location.pathname;
```

Comment:
just so you know, I don't think this will include the `?` in a href and anything after it, so e.g. in `/en-US/questions?step=1` it won't include the `?step=1`.  So if you want the whole thing you'll likely want to do `path = window.location.pathname + window.location.search`.

Reply:
Ahhh good point. I don't think those results pages ever have params like that but we should make sure we can support if we ever do.